archimedes/archimedes/runner.py

```diff
@@ -0,0 +1,314 @@
+"""
```

Comment:
This is really just a demonstration of a synchronous scheduler, suitable for locally running scripts or running in docker.  
In this version, I imagine that the vulcan web endpoint just calls this runner directly, by invoking the `archimedes:latest` image and running `poetry run archimedes-run --args --here`.  In the future, though, this logic would get folded into whatever scheduler we decided to use full time.